client/src/redux/ga-saga.js

```diff
@@ -1,9 +1,39 @@
-import { takeEvery, call, all } from 'redux-saga/effects';
+/* eslint-disable camelcase */
+import { takeEvery, call, all, select } from 'redux-saga/effects';
+import { ABTestConfig } from '../../../config/donation-settings';
 import ga from '../analytics';
+import {
+  isSignedInSelector,
+  emailSelector,
+  completionCountSelector,
+  completedChallengesSelector
+} from '../redux';
+import { emailToAVariant } from '../utils/A-B-tester';
```

Comment:
```suggestion
import { emailToABVariant } from '../utils/A-B-tester';
```
The first one reads like "email to a variant", but this reads "email to AB variant", at least to me.